hydra-data/src/main/java/com/addthis/hydra/data/query/QueryElementNode.java

```diff
@@ -188,6 +199,39 @@ public QueryElementNode parse(String tok, MutableInt nextColumn) {
         return this;
     }
 
+    /**
+     * The prefix "+[text]" or "+%[text] is interpreted as a default
+     * value. Extract the default value and remove it from
+     * the input token. Optionally "+[text|hits]" can be used to set
+     * the number of hits on the default node.
+     */
+    private String extractDefaultValue(String tok) {
```

Comment:
method is.. pretty intense. Given that it is called only once per query (well, per worker maybe?), I think you should feel free to trade performance for clarity if possible. The docs are pretty great though, so I would also be happy with just a unit test.